src/libs/ReportActionsUtils.js

```diff
@@ -40,6 +40,19 @@ function isDeletedAction(reportAction) {
     return message.length === 0 || lodashGet(message, [0, 'html']) === '';
 }
 
+/**
+ * Returns the parentReportAction if the given report is a thread.
+ *
+ * @param {Object} report
+ * @returns {Object}
+ */
+function getParentReportAction(report) {
```

Comment:
It looks like this is not returning an action, just an ID. So we should change the name/docs/returns for all of this right?

Reply:
No this is returning the action

Reply:
This is returning an action and not an ID.